### ISSUE TITLE
Hello migrating from 1.4.21 -> 1.5.1 : Product and Attributes don't show?

### ISSUE THREAD

**Issue by amitvkhajanchi** [akeneo/pim-community-dev#4331]
**I'm reporting a Bug**

Hello, I was just testing the migration from 1.4.21 -> 1.5.1.  I have followed the instruction for upgrading from 1.4 to 1.5.  Everything worked fine, except when I go back into Akeneo - I don't see my Products nor Attributes show up?   I looked at the database level and all the information is there?  Why is this not being loaded? 

The functionality of all 1.5.1 is there it seems, but my previously uploaded data is not showing?

Alternatively, what is the best way to migrate content from old version to new version?  I can export Product, Families, Attribute, Attribute Options, Category... for new install.  However, how about users, channels etc.   Its a pain to setup all these in yml file, as we used GUI to create these on live site.

[akeneo-pim-system-info_2016-04-16_10_38.txt](https://github.com/akeneo/pim-community-dev/files/222322/akeneo-pim-system-info_2016-04-16_10_38.txt)

**Comment by jmleroux** [comment]
@amitvkhajanchi 

No JS errors in development console ?

You should clear your cache, wich means:
- clear Symfony cache. The fastest way is `rm -rf app/cache/*`
- restart apache2 to avoid some APC issues
- regenerate your assets

**Comment by amitvkhajanchi** [comment]
Hi.  I tried your suggestion, then also made sure the owner of the root directory and all files are done properly, and yet it was not resolving.   Some pages showed up fine, some components didn't.  So just for kicks, I tried using different browser (was using Firefox 45.0.1, then tried Chrome) and viola- all data showed up.  Seems it was browser compatibility issue.  My OS is Ubuntu LTS 14.04 (my test machine).

**Comment by a2xchip** [comment]
@amitvkhajanchi I run all environment variant on ubuntu 14.04. And both versions are working fine with both browsers. Some times assets refresh inhibit somehow. Than i delete web/bundle directory and reinstall them all once again.

**Comment by amitvkhajanchi** [comment]
I updated my firefox and rebooted and all seems to be working fine.  Thanks for all the help! 

I did do the following though in previous steps for reference.

rm -rf app/cache/* ./web/bundles/* ./web/css/* ./web/js/*
app/console --env=prod pim:installer:assets
app/console --env=prod cache:warmup

In my case, it seemed to have been some firefox issue.  It maybe have been because I ran out of diskspace few days ago during update and had to expand my diskspace in my virtual machine.  Perhaps, the update didn't go through completely.  Either way, its now fixed.  Thanks you!
